src/main/java/org/mockbukkit/mockbukkit/block/data/CaveVineDataMock.java

```diff
@@ -0,0 +1,46 @@
+package org.mockbukkit.mockbukkit.block.data;
+
+import org.bukkit.Material;
+import org.bukkit.block.data.type.CaveVines;
+import org.jetbrains.annotations.NotNull;
+
+import static org.mockbukkit.mockbukkit.block.data.BlockDataKey.BERRIES;
+
+public class CaveVineDataMock extends AgeableDataMock implements CaveVines
+{
+
+	public CaveVineDataMock(@NotNull Material material)
+	{
+		super(material);
+	}
+
+	protected CaveVineDataMock(@NotNull CaveVineDataMock other)
+	{
+		super(other);
+	}
+
+	@Override
+	public boolean hasBerries()
+	{
+		return super.get(BERRIES);
+	}
+
+	@Override
+	public boolean isBerries()
+	{
+		return super.get(BERRIES);
+	}
```

Comment:
We don't need to implement this, since the `isBerries()` is deprecated and has a default implementation:
```java
    /**
     * Gets the value of the 'berries' property.
     *
     * @return the 'berries' value
     * @deprecated bad name, use {@link #hasBerries()}
     */
    @Deprecated
    default boolean isBerries() {
        return this.hasBerries();
    }
```